functions.php

```diff
@@ -2,10 +2,15 @@
 
 function displayDB(array $results) : string {
     $dbResult = ' ';
+    if($results === []){
+        return 'Incorrect input received.';
+    }
+
     foreach($results as $result){
     $dbResult .= '<div>'. '<img src="images/' . $result['image-link'] . '"' . ' alt="a painting of '
                . $result['painting-name'] .' " /><p>'
             . $result['artist'] . '<br>' . $result['painting-name'] . '<br>' . $result['year-made'] .  '</p></div>';
     }
     return $dbResult;
```

Comment:
All looks good and ticks the boxes! Only minor thing I would suggest is to neaten up this function a bit by breaking up each $result onto a new line.